source/auth/auth.rst

```diff
@@ -5,8 +5,20 @@
 Driver Authentication
 =====================
 
+<<<<<<< HEAD
+=======
+:Spec: 100
+:Spec Version: 1.12.0
+:Title: Driver Authentication
+:Author: Craig Wilson, David Golden
+:Advisors: Andy Schwerin, Bernie Hacket, Jeff Yemin, David Golden
+>>>>>>> a1756248 (update changelog and version)
 :Status: Accepted
 :Minimum Server Version: 2.6
+<<<<<<< HEAD
+=======
+:Last Modified: 2022-07-27
+>>>>>>> a1756248 (update changelog and version)
```

Comment:
Fix merge conflicts. Much of the front matter was removed by https://jira.mongodb.org/browse/DRIVERS-2253.

Reply:
Fixed